daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-architecture.md

```diff
@@ -150,16 +150,16 @@ Because Dapr Workflows are internally implemented using actors, Dapr Workflows h
 
 The expected scalability of a workflow is determined by the following factors:
 
-* The number of machines used to host your workflow application
-* The CPU and memory resources available on the machines running workflows
-* The scalability of the state store configured for actors
-* The scalability of the actor placement service and the reminder subsystem
+- The number of machines used to host your workflow application
+- The CPU and memory resources available on the machines running workflows
+- The scalability of the state store configured for actors
+- The scalability of the actor placement service and the reminder subsystem
 
-The implementation details of the workflow code in the target application also plays a role in the scalability of individual workflow instances. Each workflow instance executes on a single node at a time, but a workflow can schedule activities and child workflows which run on other nodes. 
+The implementation details of the workflow code in the target application also plays a role in the scalability of individual workflow instances. Each workflow instance executes on a single node at a time, but a workflow can schedule activities and child workflows which run on other nodes.
 
 Workflows can also schedule these activities and child workflows to run in parallel, allowing a single workflow to potentially distribute compute tasks across all available nodes in the cluster.
 
-TODO: Diagram showing an example distribution of workflows, child-workflows, and activity tasks.
+<img src="/images/workflow-overview/workflow-actor-scale-out.png" width=800 alt="Diagram of workflow and activity actors scaled out across multiple Dapr instances"/>
```

Comment:
May be worth annotating each of the gray boxes to say that this is a node in the cluster.